CRM/Aoservicelisting/Form/ProviderApplicationForm.php

```diff
@@ -35,17 +35,16 @@ public function setDefaultValues() {
         ]);
         $primaryContact = civicrm_api3('Contact', 'getsingle', [
           'id' => $this->_loggedInContactID,
-          'return' => ['id', 'first_name', 'last_name', CERTIFICATE_NUMBER]
+          'return' => ['id', 'first_name', 'last_name', CERTIFICATE_NUMBER, REGULATED_URL]
         ]);
         $primaryContactPhone = civicrm_api3('Phone', 'getsingle', ['contact_id' => $this->_loggedInContactID, 'is_primary' => 1]);
-        $website = civicrm_api3('Website', 'get', ['contact_id' => $this->_loggedInContactID, 'url' => ['IS NOT NULL' => 1], 'sequential' => 1])['values'];
-        $regulatorUrlPresent = (!empty($website));
+        $regulatorUrlPresent = (!empty($primaryContact[REGULATED_URL]));
         $staffRowCount = $abaStaffCount = 1;
```

Comment:
I wanted to remove this line and the code uptil L86, but was hesitant. Maybe we can discuss how we can remove code that is not needed anymore?

Reply:
@Edzelopez why isn't it required any more?

Reply:
Because they no longer want the regulator URL to be saved as a website.

Reply:
@Edzelopez this line -> L86 should still be required right in that because it sets the default values for the first ABA or first Regulated staff if they are also the primary contact right?